core/model/src/main/java/org/activityinfo/model/resource/Resources.java

```diff
@@ -60,6 +60,13 @@ public static Resource fromJson(String json) {
         return fromJson(resourceObject);
     }
 
+    public static Record fromJson_(String json) {
```

Comment:
Improve naming here. For example, rename to recordFromJson() and resourceFromJson()